hammett/types.py

```diff
@@ -42,6 +42,8 @@ def __call__(
         screen: 'Screen',
         update: 'Update',
         context: 'CallbackContext[BT, UD, CD, BD]',
+        *args: tuple['Any'],
+        **kwargs: 'Any',
```

Comment:
Be more specific about the `kwargs` type. It's `dict[str, 'Any']` (used throughout the framework),

Reply:
I checked it out through sources.debian.org. Both `args` and `kwargs` are typed as `Any`, it allows avoiding some pitfalls.
@Themanwhosmellslikesugar, replace the `args` type for `Any`. @warewarewapain, reflect the change in Hammett globally.